### ISSUE TITLE
Error in starting boilerplate's pong game

### ISSUE THREAD

**Issue by DimaValetsky** [lance-gg/tinygames#3]
![изображение](https://user-images.githubusercontent.com/48615297/56293099-21362880-6131-11e9-9b40-46d1218678ca.png)
after diong all steps in guide I see this
npm ERR! lance-boilerplate@4.0.0 start: `node dist-server/main.js`

**Comment by juancjara** [comment]
Maybe you already solved it, but you have to import TwoVector from `lance-gg`, something like ```import { TwoVector } from 'lance-gg'```

**Comment by mreinstein** [comment]
good catch! This issue is already logged @ https://github.com/lance-gg/lance/issues/126 

there's also a PR waiting to go out which should fix this. Can probably close this as duplicate.
